darwin/dataset/download_manager.py

```diff
@@ -316,15 +366,16 @@ def _update_local_path(annotation: AnnotationFile, url, local_path):
 
     # we modify raw json, as internal representation does't store all the data
     with annotation.path.open() as file:
-        raw_annotation = json.load(file)
+        raw_annotation = json.loads(file.read())
 
         for slot in raw_annotation["item"]["slots"]:
             for source_file in slot["source_files"]:
                 if source_file["url"] == url:
                     source_file["local_path"] = str(local_path)
 
     with annotation.path.open(mode="w") as file:
-        json.dump(raw_annotation, file, indent=4)
+        op = json.dumps(raw_annotation, json.OPT_INDENT_2)
```

Comment:
We can't recreate 4-space indented JSON with `orjson` - it offers 2 spaces, or no spaces.  For the performance gain, this seems fair.